spec/support/final_destination_helper.rb

```diff
@@ -7,7 +7,15 @@
 
     def self.enable!
       FinalDestination.send(:remove_const, :HTTP)
-      FinalDestination.send(:const_set, :HTTP, Net::HTTP)
+
+      # At this point, `Net::HTTP` has already been patched by Webmock so we need to basically re-declare the `FinalDestination::HTTP`
+      # but inherit from the patched `Net::HTTP` class. This is to allow requests made using `FinalDestination::HTTP` to be
+      # intercepted by Webmock.
+      FinalDestination.send(
+        :const_set,
+        :HTTP,
+        Class.new(Net::HTTP) { include FinalDestination::SafeNetHTTP },
```

Comment:
This is what I mean when `Net::HTTP` has been "patched" by webmock.

https://github.com/bblimke/webmock/blob/833291d4ce2d5927a738f929db26b594bf4fa7f5/lib/webmock/http_lib_adapters/net_http.rb#L14-L19